e2e/libs/node/node.py

```diff
@@ -3,12 +3,36 @@
 import time
 import logging
 from utility.utility import apply_cr_from_yaml, get_cr
+from utility.utility import wait_for_cluster_ready
+import boto3
 
 RETRY_COUNT = 180
 RETRY_INTERVAL = 1
 
 class Node:
 
+    def __init__(self):
+        with open('/tmp/instance_mapping', 'r') as f:
+            self.mapping = yaml.safe_load(f)
+        self.aws_client = boto3.client('ec2')
```

Comment:
The implementation should be extendable to support different env in the future. Please create a github issue for this.

Reply:
Can all supported environments be listed here? Then the related issue can have a clear description and scope.

Reply:
It will depend on the implementation. Surely we can implement here but delegate the call to the corresponding infra implementation. Let u and the team to think about that.

cc @longhorn/qa

Reply:
https://github.com/longhorn/longhorn/issues/6709 created for this.